.github/workflows/scxa-solrcloud.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build and Push Image
+
+on:
+  push:
+    paths:
+      - "scxa-solrcloud/Dockerfile"
+
+jobs:
+  build:
+    name: Build and push image
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: scxa-solrcloud
+
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read upstream tag without version
+      id: gettag
+      run: echo "::set-output name=tag::$(head -n 1 scxa-solrcloud/Dockerfile | awk -F':' '{print $2}' | awk -F'-' 'BEGIN { OFS="-" } {$NF=""; print $0}')"
+
+    - name: Read internal update version
```

Comment:
You don't have the RUNNER_VERSION issue the other container has, so you should remove this step...